test/project1/source/consistent-return.brs

```diff
@@ -46,6 +46,15 @@ sub error9()
     end sub)
 end sub
 
+function error10() as integer
+    a = 1
+    if a > 0
+        throw "something wrong"
+    else
+        return 0' missing return or throw here
```

Comment:
Why would that case be an error? It's the same code as `ok7`, no?

Reply:
My bad the `return` shouldn't be there you're right

Reply:
`throw` should be also valid for not returning anything.

Reply:
Yes but it's not a void  function here.

Reply:
I know but is this OK too?
```
function ok() as void
    a = 1
    if a > 0
        throw "something wrong"
    end if
end function
```

Reply:
Oh sorry, yes this one works fine, I'll add it in the test cases